source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -146,11 +149,19 @@ OpenGLContext: class extends GpuContext {
 		}
 	}
 	update: func { this _backend swapBuffers() }
-	packToRgba: func (source: GpuImage, target: GpuImage, viewport: IntBox2D) {
+	packToRgba: func (source: GpuImage, target: GpuImage, viewport: IntBox2D, padding := 0) {
 		channels := 1
```

Comment:
Okay. Doesn't need fixing here, but it would make this prettier and could be useful elsewhere, too.